packages/markdown-docx/src/CiceroMarkToOOXML/rules.js

```diff
@@ -16,83 +16,107 @@
 
 const { sanitizeHtmlChars, titleGenerator } = require('./helpers');
 
+const headingLevels = {
+    1: { style: 'Heading1', size: 25 },
+    2: { style: 'Heading2', size: 20 },
+    3: { style: 'Heading3', size: 16 },
+    4: { style: 'Heading4', size: 16 },
+    5: { style: 'Heading5', size: 16 },
+    6: { style: 'Heading6', size: 16 },
+};
+
 /**
- * Inserts text.
+ * Wraps given value in OOXML text(<w:t>) tag.
  *
  * @param {string}  value Enclosing value of the OOXML tag
  * @returns {string} OOXML tag for the text
  */
-const TEXT_RULE = (value) => {
+const TEXT_RULE = value => {
     return `
-        <w:r>
-            <w:t xml:space="preserve">${sanitizeHtmlChars(value)}</w:t>
-        </w:r>
+      <w:t xml:space="preserve">${sanitizeHtmlChars(value)}</w:t>
     `;
 };
 
 /**
- * Inserts emphasised text.
+ * Provides the OOXML for emphasis.
  *
- * @param {string}  value  Enclosing value of the OOXML tag
- * @returns {string} OOXML tag for the emphasised text
+ * @returns {string} Emphasis properties in OOXML tag
  */
-const EMPHASIS_RULE = (value) => {
+const EMPHASIS_RULE = () => {
     return `
-        <w:r>
-            <w:rPr>
-                <w:i />
-            </w:rPr>
-            <w:t>${sanitizeHtmlChars(value)}</w:t>
-        </w:r>
+      <w:i/>
+      <w:iCs/>
     `;
 };
 
 /**
- * Inserts strong text.
+ * Generates OOXML for paragraph properties.
  *
- * @param {string}  value  Enclosing value of the OOXML tag
- * @returns {string} OOXML tag for the strong text
+ * @param {string} value OOXML properties to be wrapped
+ * @param {string} level Level of heading
+ * @returns {string} OOXML tag for paragraph properties
  */
-const STRONG_RULE = value => {
+const PARAGRAPH_PROPERTIES_RULE = (value, level = '') => {
     return `
-      <w:r>
-        <w:rPr>
-          <w:b />
-          <w:bCs />
-        </w:rPr>
-        <w:t>${sanitizeHtmlChars(value)}</w:t>
-      </w:r>
-`;
+      <w:pPr>
+        ${level ? `<w:pStyle w:val="${headingLevels[level].style}"/>` : ''}
+        ${value}
+      </w:pPr>
+    `;
```

Comment:
This is supposed to be for heading, right? Let's follow CiceroMark conventions only as levels of a paragraph will be obscure for others.

Reply:
@algomaster99 Yes this is for heading. 
> Let's follow CiceroMark conventions only as levels of a paragraph will be obscure for others.

I did not understand this part.

Reply:
These are not paragraph properties per se. They are heading properties so `PARAGRAPH_PROPERTIES_RULE` is not a good name.

Reply:
Will `Heading_Rule` be good?

Reply:
These are heading properties, right? You can add the properties too.

Reply:
@algomaster99 
Just saw a thing regarding these rules. The test coverage shows that they are not being used anywhere in the code. Despite this our document using ooxml is working fine. So should I remove it or fix it or let it remain as it is currently?

Reply:
If you would have deleted the test for heading, then this would be possible.

Reply:
@algomaster99 
The test for heading are not deleted.

Reply:
Look at it once then. This block of code must be executed as well test for all types of headings.

Reply:
@algomaster99 
Yeah it was being executed. I pointed out the wrong code. Apologies.